security/src/main/java/com/networknt/security/JwtVerifier.java

```diff
@@ -375,6 +372,21 @@ private boolean isJwtAudienceValid(JwtClaims claims, String audience) throws Mal
         return claims.getAudience().contains(audience);
     }
 
+    private boolean isJwtAudienceValid(JwtClaims claims, List<String> jwkServiceIds) throws MalformedClaimException {
+        boolean r = false; // the initial value is false.
+        for(String serviceId: jwkServiceIds) {
+            if(audienceMap != null && audienceMap.size() > 0) {
```

Comment:
@DiogoFKT Based on your suggestions, I have added the logic to handle the case that none of the serviceId has an audience configured. I have tested with the light gateway, and it will pass the validation in this case. Please review and let me know if there are any issues. Thanks.